tags/rules.tag

```diff
@@ -0,0 +1,6 @@
+type: text
+
+---
+
+> 2. Only use the support channels if you need help. **Please do not ping, DM, or project staff if you need support.** We all have lives outside of Geyser and it may take some time for us to respond. With that being said, please don't ask for support in non-support channels as well.
+Please read the rules in <#613195277938982953>
```

Comment:
We should probably import all of them before we merge this

Reply:
Yes, that will be done before this is merged and made live